ScreenControl_Web/src/Connection/ScreenControlServiceTypes.ts

```diff
@@ -53,6 +53,37 @@ export class ConfigState
     }
 }
 
+// class: ConfigRequest
+// Used to request the current state of the configuration on the Service. This is received as
+// a <ConfigState> which should be linked to a <ConfigStateCallback> via requestID to make
+// use of the data received.
+export class ConfigRequest
```

Comment:
Same deal here as in WebsocketReciever.cs: can we distinguish the config _change_ request and the config _state_ requests?